modules/base/rotation/timelinerotation.cpp

```diff
@@ -68,9 +68,7 @@ TimelineRotation::TimelineRotation(const ghoul::Dictionary& dictionary)
         ghoul::mm_unique_ptr<Rotation> rotation = Rotation::createFromDictionary(
             kf.second
         );
-        if (rotation) {
-            _timeline.addKeyframe(t, std::move(rotation));
-        }
+        _timeline.addKeyframe(t, std::move(rotation));
```

Comment:
Do we want to add it even if it couldn't be created?  Maybe this is not an issue and is caught somewhere higher up

Reply:
This was a remnant from how the `createFromDictionary` function worked before.  In the past, it returned a `nullptr` if the creation of the object failed.  By now all of those functions are instead throwing an exception if the creation failed, meaning that if we get to the if-statement we didn't get an exception, meaning that the pointer is going to be `!= nullptr`